### DISCUSSION TITLE
Forward node specific Suricata rules.

### DISCUSSION THREAD

**Discussion by LucasCortooms** [Security-Onion-Solutions/securityonion#8894] [Unsupported Versions] [answered]
Hello all,

I have an idea that I would like to implement but I don't know if this is possible. So to start, I have a distributed deployment of Security-union. I have multiple forward nodes at different locations that send data to a central manager node. I want to add suricata rules that only apply to one of the many forward nodes. I'd like to implement this because I want to create custom rules for only one specific site. It is possible to filter to some extent in a Suricata rule, for example filtering by subnets. This is not an option in my case because some sites have the same subnet. I'm trying to figure out if I'm missing something or if there are other options available. Is there a possibility to filter on the hostname or IP address of the forward node in a Suricata rule?

Thanks in advance!

**Comment by dougburks** [comment, marked as answer]
One option might be to run the rules on all forward nodes but configure a threshold or suppression that only applies to specific sensors:
https://docs.securityonion.net/en/2.3/managing-alerts.html#threshold

**Reply by LucasCortooms** [reply]
Thank you for the quick response. This seems like it could work I will be looking into this!
